lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -225,8 +237,7 @@ def get_tick_iterators(self, axes):
 
             def _f(locs, labels):
                 for x, l in zip(locs, labels):
```

Comment:
Optional, but
```suggestion
                for loc, label in zip(locs, labels):
```
would be a bit clearer. in `self._to_xy(x, const=self._pos)` *x* can be a y-coordinate depending on `self._pos`, so better use `self._to_xy(loc, const=self._pos)`.

Reply:
sure